encoders/firebase-decoders-json/src/main/java/com/google/firebase/decoders/TypeTokenContainer.java

```diff
@@ -41,4 +42,48 @@ public TypeTokenContainer(@NonNull TypeToken[] typeTokens) {
     TypeToken<T> typeToken = (TypeToken<T>) typeTokens[index];
     return typeToken;
   }
+
+  @Override
+  public int hashCode() {
```

Comment:
I think this whole method could be just one line:
```java
return Arrays.hashCode(typeTokens);
```
wdyt?

Reply:
Good call!!